src/workloads/c2c/MongosyncScripts.yml

```diff
@@ -0,0 +1,96 @@
+SchemaVersion: 2018-07-01
+
+Description: |
+  This is a collection of helper scripts used to communicate with mongosync. To use them,
+  use LoadConfig and load the script you would like to use in your actor's phase.
+  For example:
+  - Name: StartMongosync
+    Type: ExternalScriptRunner
+    Threads: 1
+    Phases:
+      - LoadConfig:
+        Path: ./MongosyncScripts.yml
+        Key: StartMongosync
+
+  Note that the polling scripts start with set -eou pipefail so that any
+  subcommand run during the script failing will cause the entire script to fail.
+  This way if mongosync crashes during the test, it will stop the test execution
+  immediately and genny will bubble up the error and fail the test
+
+  For curl commands we pass -s and -S so that an errors are still reported (-S)
+  but download progress stats are not logged to stderr
+
+StartMongosync:
+  Repeat: 1
+  Command: sh
+  MetricsName: StartMongosync
+  Script: |
+    curl -s -S -X POST http://10.2.0.160:27182/api/v1/start \
```

Comment:
I wish there was a way to pipe in the IP address. That looks really brittle, and it means we can't use this workload as is unless we use our infrastructure provisioning and ensure we have the correct IP. That said, I'm not sure how we would get it in. There is some ability to pass in variables to Genny, but likely painful and ugly.

Reply:
Yeah I agree. I think this is going to get more complicated with adding multiple C2C urls also, so I have started thinking about how to handle that.

One idea I had is a top-level config in genny like:
```
EnvironmentVariables:
   key1: a
   key2: b
```

Then we could pass it in from DSI via genny's overrides.yml, and for the script invocation append all the key/value pairs under environment variables as `key1="a" key2="b" sh temporary_path.sh`.

What are your thoughts?

Reply:
Oh that could work. I would just encourage you to not get too clever in bash. It doesn't end well :)